csr_check.go

```diff
@@ -151,7 +152,7 @@ func authorizeCSR(config ClusterMachineApproverConfig, machines []v1beta1.Machin
 		var foundSan bool
 		for _, addr := range targetMachine.Status.Addresses {
 			switch addr.Type {
-			case corev1.NodeInternalDNS, corev1.NodeExternalDNS, corev1.NodeHostName:
+			case corev1.NodeInternalDNS, corev1.NodeExternalDNS, corev1.NodeHostName, altInternalDNS:
```

Comment:
Instead of using the custom `AltInternalDNS`, why not have 2 entries with a type of `corev1.NodeInternalDNS` and then make sure this code doesn't stop checking on the first one?

Reply:
That sounds good. I don't like introducing yet another key.

Reply:
So it looks like it already does that ... will close this PR and make the change over there. Thx!